### DISCUSSION TITLE
Fetching more than 100 items in nested structure

### DISCUSSION THREAD

**Discussion by andriusign** [directus/directus#6522] [Archive]
Directus by default returns 100 items unless specified otherwise with limit param. This is okay when fetching single type of items.

How can I fetch more than 100 items in nested object when? E.g. Article has comments relationship, fetching items/articles only fetches 100 max of article.comments in nested object.

**Comment by philleepflorence** [comment]
Hello, hope this helps answer your question:

### Setting query parameters on a nested relational dataset
[Deep - API Reference](https://docs.directus.io/reference/api/query/#deep)
